chapters/usage-profile.md

```diff
@@ -0,0 +1,326 @@
+      </Usage>
+    </UsageScope>
+  ...
+
+```
+
+## 31.2 Usage Scope Comment field <a name="31.2"></a>
+
+### 31.2.1 Description
+
+This field provides a place for the SPDX document creator to record any general comments about the "usage scope".
+
+Table 31.2 — Metadata for the usage scope comment field
+
+| Attribute | Value |
+| --------- | ----- |
+| Required | No |
+| Cardinality | 0..1 |
+| Format | Free form text that may span multiple lines, refers only to the immediately preceding usagescope.  |
+
+
+### 31.2.2 Intent
+
+To provide additional context. For example, describing prerequisite assumptions of usage scope by the package supplier.
+
+### 31.2.3 Examples
+
+EXAMPLE 1 Tag: `UsageScopeComment:`
+
+```text
+UsageScope: SPDXRef-a_specific_binary_verNNN: shipped for consumer product/ medical system / automotive system
+UsageScopeComment: A_specific_binary_verNNN is compiled and velified as production level quality
+```
+
+EXAMPLE 2 RDF: Property `rdfs:comment` in class `spdx:UsageScope`
+
+```text
+<UsageScope>
+         <rdfs:comment> A_specific_binary_verNNN is compiled and velified as production level quality</rdfs:comment>
+</UsageScope>
+```
+
+
+## 31.3 UsageScope SPDX identifier field <a name="31.3"></a>
+
+### 31.3.1 Description
+
+Uniquely identify any element in an SPDX document which may be referenced by other elements. These may be referenced internally and externally with the addition of the SPDX document identifier. The metadata for the SPDX identifier field is shown in Table NN.
+
+**Table 31.3 — Metadata for the UsageScope SPDX identifier field**
+
+| Attribute | Value |
+| --------- | ----- |
+| Required | No |
+| Cardinality | 1..1 |
+| Format | "SPDXRef-"`[idstring]` <br> where `[idstring]` is a unique string containing letters, numbers, `.`, and/or `-`. |
+
+### 31.3.2 Intent
+
+There may be several cases of the same package within an SPDX document. Each element needs to be able to be referred to uniquely so that relationships between elements can be clearly articulated.
+
+### 31.3.3 Examples
+
+EXAMPLE 1 Tag: `UsageScopeSPDXID:`
+
+```text
+UsageScopeSPDXID: SPDXRef-1
+```
+
+EXAMPLE 2 RDF: The URI for the element will follow the form:
+
+```text
+[SPDX document namespace]#[SPDX identifier]
+```
+
+See [6.5](document-creation-information.md#6.5) for the definition of the SPDX document namespace and [6.3](document-creation-information.md#6.3) for the definition of the SPDX identifier
+
+Using `xml:base`:
+
+```text
+<rdf:RDF xml:base="http://acme.com/spdxdocs/acmeproj/v1.2/1BE2A4FF-5F1A-48D3-8483-28A9B0349A1B">
+    ...
+    <UsageScope rdf:about="#SPDXRef-1">
+    ...
+    </UsageScope>
+</rdf:RDF>
+```
+
+Using document URI:
+
+```text
+<UsageScope rdf:about="http://acme.com/spdxdocs/acmeproj/v1.2/1BE2A4FF-5F1A-48D3-8483-28A9B0349A1B#SPDXRef-1">
+    ...
+</UsageScope>
+```
+
+## 31.4 Package Release Date field <a name="31.4"></a>
```

Comment:
Would it be appropriate for this field to be moved into the [Package Information section](https://spdx.github.io/spdx-spec/package-information/), rather than kept here?

This seems to be metadata about the package, and even if it's useful for a Usage Information section, it seems like it would belong as an optional field in the Package Information section along with the other Package-specific data.